empress/tools.py

```diff
@@ -263,7 +263,10 @@ def match_inputs(
             raise DataMatchingError(
                 "The feature table contains features that aren't present as "
                 "tips in the tree. You can override this error by using the "
-                "--p-filter-missing-features flag."
+                "--p-filter-missing-features flag for the QIIME 2 version of "
++               "EMPress, or the --filter-missing-features flag for the "
++               "standalone version of EMPress."
```

Comment:
With strings inside a parenthesis, the `+` operators are not needed.

```suggestion
                "EMPress, or the --filter-missing-features flag for the "
                "standalone version of EMPress."
```